client/index.js

```diff
@@ -53,3 +53,7 @@ render(
   </CookiesProvider>,
   document.getElementById("root"),
 );
+
+if (module && module.hot) {
+  module.hot.accept();
+}
```

Comment:
Hi,

Please add a line at the end of the file, notice all the files have a blank line in the end!
**Why?** Because some old compliers wouldn't compile the last line, so due to the legacy compilers it has become a developer practice to do this.